docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>It feels like you are saying a lot here but you aren't showing the evidence for it until later. It is okay to make a statement and show the evidence inline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>